backend/libs/gen_secret_key.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from random import choice
+from secrets import choice
 
 
 def gen_secret_key(r):
```

Comment:
Minor: as this PR aims at increasing the security of our application, what about replacing choice(...) with [secrets.token_urlsafe](https://docs.python.org/3/library/secrets.html#secrets.token_urlsafe)? It seems safer as more characters are used to define the secret.

Reply:
I tried to be as minimal as possible, but sure, I could use this other method.

One question though : is it important that the secret is exactly 50 chars long or can it be any size (must be long enough of course). Since token_urlsafe takes a number of bytes as input, it's just to know if I need to write `token_urlsafe(50)[:50]` or just `token_urlsafe(40)` (doc says it's about 1.3 chars per byte).

Reply:
I'm not sure it's worth it changing this function.As you said, it's good to add as minimal change as possible. We could see later how to improve this code.

Reply:
Sorry for the noise :)